src/server.js

```diff
@@ -1,11 +1,18 @@
 const app = require('./app')
 const connectDB = require('../db/connect')
+const http = require('http')
+const intiatSocket = require('./socket')
```

Comment:
I think this should be spelled as initiateSocket. If you update this here, be sure to update all other references.

Reply:
Will do Jacob, thanks alot for the review.